app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -111,11 +109,12 @@ def authorization_params
 
     def pre_validate_authorize_form
       result = @authorize_form.submit
+      analytics_attributes = result.to_h.except(:redirect_uri, :code_digest)
+      analytics_attributes[:user_fully_authenticated] = user_fully_authenticated?
+      analytics_attributes[:referer] = request.referer
+      analytics.openid_connect_request_authorization(**analytics_attributes)
```

Comment:
potentially inline this?
```suggestion
      analytics.openid_connect_request_authorization(
        **result.to_h.except(:redirect_uri, :code_digest).merge(
          user_fully_authenticated: user_fully_authenticated?,
          referer: request.referer,
        ),
      )
```